roles/vgpu/templates/nvidia-sriov.service.j2

```diff
@@ -6,12 +6,15 @@ After=local-fs.target {{ vgpu_systemd_device[vgpu_definition.pci_address] }}
 Wants={{ vgpu_systemd_device[vgpu_definition.pci_address] }}
 
 [Service]
+Restart=on-failure
+RestartSec=30
 Type=oneshot
 User=root
 # NOTE(wszumski): There is a race in the driver initialization where if we run this too early, then
 # the mdev_support_devices entry doesn't show up in sysfs. I was unable to get this to show up again
 # without a reboot.
 ExecStartPre=/bin/sleep 5
+ExecStart=/bin/bash -c "echo '{{ vgpu_definition.pci_address }}' > /sys/bus/pci/drivers/nvidia/bind || true"
```

Comment:
Would be nice to have a short comment to describe why this is a necessary addition.

Reply:
Good suggestion, I've added a comment.